tests/phpstan-next.txt

```diff
@@ -0,0 +1,93 @@
+# List of files, which are not yet passing PHPStan level 6 https://phpstan.org/user-guide/rule-levels
```

Comment:
The idea is to remove lines from that list when pull requests are claiming an improved PHPStan level for some specific files.
Example: https://github.com/FreshRSS/FreshRSS/pull/5230/commits/a5cc7b20f6b05ceae73a90dc418345f7f0dede6f